lisa/nic.py

```diff
@@ -29,23 +29,34 @@ class NicInfo:
 
     def __init__(
         self,
-        upper: str,
+        name: str,
+        upper: str = "",
         lower: str = "",
         pci_slot: str = "",
+        is_upper: bool = True,
+        driver_sysfs_path: Optional[PurePosixPath] = None,
+        expected_no_vf: Optional[bool] = None,
```

Comment:
Consider both PF and VF. what's the purpose here? Is it to identify only VF, or it's to identify Accelerated Networking?

Reply:
The name should be like `is_vf` or `is_synthetic`. Avoid to use `no`, `not` in variable name.

Reply:
The expected value shouldn't be saved in nics or features. The expected values should be managed in test case level.

Reply:
I think it's misused in two purposes. One is to set expected value on nic in `wait_for_sriov_enabled`, it should be pass in from the method. The other is to check if it's synthetic or pci, so it should be renamed to `is_pci`.

Reply:
you mean in `wait_for_sriov_enabled`. I should not use expected_no_vf, instead, I need a parameter in the method to tell whether we need check the VF nic or not?